src/activities/railroad/railroad.js

```diff
@@ -0,0 +1,90 @@
+/* GCompris - railroad.js
+ *
+ * Copyright (C) 2016 Irshaad Ali <irshaadali14@gmail.com>
+ *
+ * Authors:
+ *   Bruno Coudoin <bruno.coudoin@gcompris.net> (GTK+ version)
+ *   Irshaad Ali <irshaadali14@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+.pragma library
+.import QtQuick 2.0 as Quick
+
+var currentLevel = 0
+var numberOfLevel = 3
+var items
+
+var backgroundImage = "railroad-bg.svg"
+
+
+var trainParts = [
+            ["loco1.svg"],
```

Comment:
Are there really svg images or png in svg. If this is just png in svg, it is best to keep the image in the png format on disk.